hack/make/tests/e2e.mk

```diff
@@ -8,3 +8,7 @@ test/e2e/activegate:
 ## Runs CloudNative e2e test only
 test/e2e/cloudnative:
 	go test -v -tags e2e -count=1 -failfast ./test/cloudnative
+
+## Runs Application Monitoring e2e test only
+test/e2e/appmon:
```

Comment:
nitpick: I don't think this is the right place to shorten `applicationmonitoring` to `appmon`

Reply:
renamed the target